src/main/java/com/odysseusinc/arachne/executionengine/util/AnalisysUtils.java

```diff
@@ -149,4 +152,16 @@ private static void decompressToDir(File parent, List<MultipartFile> files) thro
         }
         return CommonFileUtils.getFSResources(resultFiles);
     }
+
+    public static String replaceBigQueryKeyPath(String connectionString, String replacement) {
+
+        return connectionString.replaceFirst(BQ_KEYPATH_REGEX,
```

Comment:
What will the line replace if you remove the param in WebAPI? - https://github.com/OHDSI/WebAPI/blob/ac8a669e91aab5cb1f017beb1f64734a54b13e18/src/main/java/org/ohdsi/webapi/service/SourceService.java#L280

Which leads to lack of the key parameter and failed login